pulp_2to3_migrate/app/models/repository.py

```diff
@@ -0,0 +1,123 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from pulpcore.app.models import (  # it has to be imported directly from pulpcore see #5353
+    Remote,
+    Publisher,
+)
+from pulpcore.plugin.models import (
+    Model,
+    BaseDistribution,
+    RepositoryVersion,
+    Publication,
+)
+
+
+class Pulp2Repository(Model):
+    """
+    Information about Pulp 2 repository.
+
+    Fields:
+        pulp2_repo_id (models.CharField): Repository ID in Pulp 2
+        pulp2_object_id (models.CharField): Object id of a repository in Pulp 2
+        pulp2_last_unit_added (models.DateTimeField): Last time a unit was added to
+            a repository in Pulp 2
+        pulp2_last_unit_removed (models.DateTimeField): Last time a unit was removed from
+            a repository in Pulp 2
+
+    Relations:
+        pulp3_repository_version (models.OneToOneField): Pulp 3 repository version which Pulp 2
+            repository was migrated to
+    """
+    pulp2_repo_id = models.CharField(max_length=255)  # in Pulp2 it might be longer?
+    pulp2_object_id = models.CharField(max_length=255)
```

Comment:
what is the reason having both object_id and repo_id? can we refer only to repo_id?

Reply:
Since we can run a migration multiple times, it's possible that a repo with a specific repo_id can be removed and then created again and it will be a completely different repo. Migration plugin needs to distinguish between those.